src/main/java/seedu/address/storage/StorageManager.java

```diff
@@ -108,4 +111,32 @@ public void saveContactList(ReadOnlyContactList contactList, Path filePath) thro
         logger.fine("Attempting to write to data file: " + filePath);
         contactListStorage.saveContactList(contactList, filePath);
     }
+
+    // ================ TodoList methods ==============================
+    @Override
+    public Path getTodoListFilePath() {
+        return todoListStorage.getTodoListFilePath();
+    }
+
+    @Override
+    public Optional<ReadOnlyTodoList> readTodoList() throws DataConversionException, IOException {
+        return readTodoList(todoListStorage.getTodoListFilePath());
+    }
+
+    @Override
+    public Optional<ReadOnlyTodoList> readTodoList(Path filePath) throws DataConversionException, IOException {
+        logger.fine("Attempting to read data from file: " + filePath);
+        return todoListStorage.readTodoList(filePath);
+    }
+
+    @Override
+    public void saveTodoList(ReadOnlyTodoList todoList) throws IOException {
+        saveTodoList(todoList, todoListStorage.getTodoListFilePath());
+    }
+
+    @Override
+    public void saveTodoList(ReadOnlyTodoList todoList, Path filePath) throws IOException {
+        logger.fine("Attempting to write to data file: " + filePath);
```

Comment:
The comment from above could be useful here as well, we could refine the logger message to be something more specific and meaningful.